src/Cron_Event_Command.php

```diff
@@ -441,26 +444,28 @@ private static function interval( $since ) {
 
 		// step one: the first chunk
 		for ( $i = 0, $j = count( $chunks ); $i < $j; $i++ ) {
-			$seconds = $chunks[$i][0];
-			$name = $chunks[$i][1];
+			$seconds = $chunks[ $i ][0];
+			$name    = $chunks[ $i ][1];
 
 			// finding the biggest chunk (if the chunk fits, break)
-			if ( ( $count = floor( $since / $seconds ) ) != 0 ){
+			$count = floor( $since / $seconds );
+			if ( floatval( 0 ) !== $count ) {
 				break;
 			}
 		}
 
 		// set output var
-		$output = sprintf( \_n( $name[0], $name[1], $count ), $count );
+		$output = sprintf( '%d %s', $count, Utils\pluralize( $name, absint( $count ) ) );
```

Comment:
If `pluralize()` comes from `WP_CLI\Utils`, the function call does not need the `Utils\` part as you've added the `use` statement.

Reply:
We would have to import the function to use `pluralize ` directly. else it won't work, else it will result in undefined.

Reply:
Yes, of course, importing functions is PHP 5.6+, so can't be done yet with the current 5.4 minimum requirement, so :+1: all good.